test/k8sT/Updates.go

```diff
@@ -140,10 +140,15 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 		By("Installing kube-dns")
 		_ = kubectl.Apply(helpers.DNSDeployment())
 
-		// Deploy the etcd operator
-		By("Deploying etcd-operator")
-		err = kubectl.DeployETCDOperator()
-		Expect(err).To(BeNil(), "Unable to deploy etcd operator")
+		switch oldVersion {
```

Comment:
@tgraf This doesn't matter, etcd-operator is just a pod like all the others, the fact it's not used in `1.0-1.2` should be irrelevant for the upgrade test. This will make the upgrade test to fail as, when upgrading to 1.3/master, there isn't a etcd cluster available so 1.3 will never be ready.

Reply:
Yes, I want to test that v1.2 comes up clean first and then involve the ETCD operator again.